src/server/views/layout.html

```diff
@@ -45,7 +44,7 @@
       </a>
     
     {% if envTag %}
```

Comment:
Do we need `envTag` everywhere?

```suggestion
    {% if config.env !== "prod" %}
```